### PR TITLE
Debug logging docs

### PR DESCRIPTION
**Pull request by ashleywang1**

### CODE REVIEW

File: docs/content/observability/ports.md [hunk 1]
```diff
@@ -1,20 +1,46 @@
 ---
-title: Stats and Admin Ports
+title: Debug Logging and Stats
 weight: 5
-description: Useful ports to know for viewing stats and admin config
+description: Useful patterns to know for debug logging, viewing stats and admin config
 ---
```
Comment (ned1313) [inline docs/content/observability/ports.md]: We should include a summary paragraph here that sets the stage for this document.

File: docs/content/security/auth/oauth/google/_index.md [hunk 2]
```diff
@@ -180,6 +180,16 @@ If you provide your Google credentials, Gloo should redirect you to the main pag
 
 ![Pet Clinic app homepage](./../petclinic-home.png)
 
+### Logging
+
+If Gloo is running on kubernetes, the extauth server logs can be viewed with:
+```
+k logs -n gloo-system deploy/extauth -f
```
Comment (ned1313) [inline docs/content/security/auth/oauth/google/_index.md]: is k aliased to kubectl?

Comment (arianaw66) [reply docs/content/security/auth/oauth/google/_index.md]: Only locally- fixed

**Pull request merged**